pkg/kubectl/cmd/annotate.go

```diff
@@ -59,36 +59,36 @@ type AnnotateOptions struct {
 }
 
 var (
-	annotate_long = dedent.Dedent(`
+	annotate_long = templates.LongDesc(`
 		Update the annotations on one or more resources.
 
-		An annotation is a key/value pair that can hold larger (compared to a label), and possibly not human-readable, data.
-		It is intended to store non-identifying auxiliary data, especially data manipulated by tools and system extensions.
-		If --overwrite is true, then existing annotations can be overwritten, otherwise attempting to overwrite an annotation will result in an error.
-		If --resource-version is specified, then updates will use this resource version, otherwise the existing resource-version will be used.
+		* An annotation is a key/value pair that can hold larger (compared to a label), and possibly not human-readable, data.
```

Comment:
It's list items. Not strictly required, but if we don't use list items here we have to either give each one its own paragraph, or make it all part of a single paragraph (no line wrapping). Markdown unwraps single line breaks.